docs/Deploying/Configuration.md

```diff
@@ -2714,19 +2702,20 @@ The content of the PEM-encoded private key for the Simple Certificate Enrollment
       -----END RSA PRIVATE KEY-----
   ```
 
-##### apple_scep_challenge
+##### mdm_apple.scep.challenge
```

Comment:
We should standardize this. It is confusing that there is `mdm.apple_scep_key` and `mdm_apple.scep.challenge`.